app/config/github.yml

```diff
@@ -34,13 +39,15 @@ parameters:
         symfony/symfony-docs:
             subscribers:
                 - app.subscriber.status_change_by_comment_subscriber
+                - app.subscriber.status_change_on_push_subscriber
```

Comment:
> I think we should just enable this on the docs for now to see how it works. We would need to get the core team to agree to this, and I really want to keep Carson from doing anything "annoying". On the docs, we're better guinea pigs :)

Quote from @weaverryan in https://github.com/carsonbot/carsonbot/pull/38#discussion_r64154578

😉